android/app/src/gps/res/values/strings.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
   <!-- Shown on app info, app switcher, etc -->
-  <string name="app_name" translatable="false">PathCheck GPS</string>
+  <string name="app_name" translatable="false">PathCheck BT</string>
   <!-- Shown below icon on launcher only -->
-  <string name="app_name_short" translatable="false">PathCheck GPS</string>
+  <string name="app_name_short" translatable="false">Path Check BT</string>
```

Comment:
Looks like this file just got overridden to an old version. Can we revert this?

Reply:
hmm... this should have gotten pulled into Lokalise. will look into this.